src/controllers/articles.controller.js

```diff
@@ -25,17 +26,11 @@ class Articles {
   static async createArticles(req, res) {
     const userId = req.auth.id;
     const findUser = await Userservice.getOneUser(userId);
-    let images = req.files;
-    images = await Promise.all(
-      images.map(async (file) => {
-        const { secure_url } = await cloudinary.v2.uploader.upload(file.path);
-        return secure_url;
-      })
-    );
+    const images = await cloudinaryHelper(req.files);
 
     if (findUser) {
       const { title } = req.body;
-      const tags = req.body.taglist.split(' ');
+      const tags = req.body.taglist ? req.body.taglist.split(' ') : [];
```

Comment:
Is this part of the taglists you're removing?

Reply:
Yes, that line should be removed also.

Reply:
I worked on this feedback.